plugins/main/bridge/bridge_test.go

```diff
@@ -171,6 +180,7 @@ var counter uint
 // arguments for a test case.
 func (tc testCase) createCmdArgs(targetNS ns.NetNS, dataDir string) *skel.CmdArgs {
 	conf := tc.netConfJSON(dataDir)
+	fmt.Println(conf)
```

Comment:
did you intend to leave this in the `testCase`?

Reply:
Nope my bad sorry